python/mxnet/_ffi/runtime_ctypes.py

```diff
@@ -0,0 +1,30 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+"""
+Adapted from incubator-tvm/python/tvm/_ffi/runtime_ctypes.py
+Common runtime ctypes.
+"""
+# pylint: disable=invalid-name
+from __future__ import absolute_import
```

Comment:
No need. Please clean up the rest of the files accordingly

Reply:
Removed all `from __future__ import absolute_import` in this pr.